core/common/src/main/java/alluxio/Constants.java

```diff
@@ -41,18 +41,19 @@
 
   public static final String EXTENSION_JAR = ".jar";
 
-  public static final String LS_FORMAT_PERMISSION = "%-15s";
-  public static final String LS_FORMAT_FILE_SIZE = "%-15s";
-  public static final String LS_FORMAT_CREATE_TIME = "%-25s";
-  public static final String LS_FORMAT_FILE_TYPE = "%-15s";
+  public static final String LS_FORMAT_PERMISSION = "%-11s";
```

Comment:
since these constants are only used for the `ls` shell command and its test, do you think it makes sense to move them to `LsCommand.java`?

Reply:
Yes, agree. I will make the change

Reply:
fixed